src/components/Explore/TokenDetails/FooterBalanceSummary.tsx

```diff
@@ -0,0 +1,184 @@
+import { useToken } from 'hooks/Tokens'
+import { useNetworkTokenBalances } from 'hooks/useNetworkTokenBalances'
+import { useState } from 'react'
+import { AlertTriangle } from 'react-feather'
+import styled from 'styled-components/macro'
+
+import { SMALLEST_MOBILE_MEDIA_BREAKPOINT } from '../constants'
+import { LoadingBubble } from '../loading'
+
+const BalanceFooter = styled.div`
+  height: fit-content;
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  border-radius: 20px 20px 0px 0px;
+  padding: 12px 16px;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 20px;
+  width: 100%;
+  color: ${({ theme }) => theme.textSecondary};
+  position: fixed;
+  left: 0;
+  bottom: 56px;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+`
+const BalanceValue = styled.div`
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  gap: 8px;
+`
+const BalanceTotal = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  color: ${({ theme }) => theme.textPrimary};
+`
+const BalanceInfo = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+`
+const FakeFooterNavBar = styled.div`
+  position: fixed;
+  bottom: 0px;
+  left: 0px;
+  background-color: ${({ theme }) => theme.backgroundBackdrop};
+  height: 56px;
```

Comment:
is this related to https://github.com/Uniswap/interface/pull/4194/files#diff-a92c5018623f57e640137b5b68f3c081ef44909861486fb2102aee0dce534df7R23?

Reply:
ya exactly -- it's basically a placeholder for the amount of space the nav bar will end up taking up

Reply:
if its related... use a variable